pages/helpers.ts

```diff
@@ -0,0 +1,9 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+export function readData<T extends object>(json_name: string): T[] {
```

Comment:
Small nit that I'll try to move into a lint rule — JS is typically camelCase land 🐫!

```suggestion
export function readData<T extends object>(jsonName: string): T[] {
```

Reply:
Changed!